packages/engine/Source/Scene/SpatialNode.js

```diff
@@ -34,6 +34,11 @@ function SpatialNode(level, x, y, z, parent, shape, voxelDimensions) {
   this.y = y;
   this.z = z;
 
+  /**
+   * @ignore
```

Comment:
I see we already use `ignore` in the class already, but why is this used over the `@private` tag which is more typical in the rest of the codebase?